packages/hub/tasks/notify-merchant-claim.ts

```diff
@@ -53,10 +57,27 @@ export default class NotifyMerchantClaim {
       throw new Error(`Subgraph did not return information for merchant claim with transaction hash: "${payload}"`);
     }
 
+    let merchantName = '';
+
+    try {
+      if (result.merchantSafe.infoDid) {
+        let merchantInfo = await this.merchantInfo.getMerchantInfo(result.merchantSafe.infoDid);
+
+        if (merchantInfo) {
+          merchantName = ` ${merchantInfo.name}`;
+        }
+      }
+    } catch (e) {
+      // Silently ignore and just show notification without merchant name
+    }
+
     let token = result.token.symbol;
     let notifiedAddress = result.merchantSafe.merchant.id;
     let amountInWei = result.amount;
-    let message = `You just claimed ${Web3.utils.fromWei(amountInWei)} ${token} from your business account`;
+
+    let message = `You just claimed ${Web3.utils.fromWei(
+      amountInWei
+    )} ${token} from your${merchantName} business account`;
```

Comment:
I think this reads better than `Revenue deposited into account for [Merchant name].` in CS-2617. Will ask in the issue.